backend/src/make_relationships.py

```diff
@@ -47,8 +48,14 @@ def create_source_chunk_entity_relationship(source_file_name :str,
     else:
         graph.query('CREATE(c:Chunk {id:"'+ chunk_uuid+'"})' + update_chunk_node_prop.format(chunk.page_content))
 
-    logging.info("make relationship between chunk node and source node")
-    graph.query('MATCH(s:Source {'+source_node.format(source_file_name)+'}) ,(c:Chunk {'+chunk_node_id_set.format(chunk_uuid)+'}) CREATE (s)-[:HAS_CHILD]->(c)')
+    logging.info("make PART_OF relationship between chunk node and document node")
+    graph.query('MATCH(d:Document {'+source_node.format(source_file_name)+'}) ,(c:Chunk {'+chunk_node_id_set.format(chunk_uuid)+'}) CREATE (c)-[:PART_OF]->(d)')
```

Comment:
in general we should use neo4j named query parameters, like `$file` or `$id` or `$source`, if possible and avoid 
not to change in this PR but going forward. You can pass them to the query execution as dict.